docs/reference/requirement-sets/excel-api-1-4-requirement-set.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Excel JavaScript API requirement set 1.44
+description: 'Details about the ExcelApi 1.4 requirement set'
+ms.date: 07/11/2019
+ms.prod: excel
+localization_priority: Normal
+---
+
+# What's new in Excel JavaScript API 1.4
+
+The following are the new additions to the Excel JavaScript APIs in requirement set 1.4.
+
+## Named item add and new properties
+
+New properties:
+
+* `comment`
+* `scope` worksheet or workbook scoped items
+* `worksheet` returns the worksheet on which the named item is scoped to.
+
+New methods:
+
+* `add(name: string, reference: Range or string, comment: string)`Adds a new name to the collection of the given scope.
+* `addFormulaLocal(name: string, formula: string, comment: string)` Adds a new name to the collection of the given scope using the user's locale for the formula.
+
+## Settings API in the Excel namespace
+
+The [Setting](/javascript/api/excel/excel.setting) object represents a key:value pair for a setting persisted to the document. The functionality of `Excel.Setting` is equivalent to `Office.Settings`, but uses the batched API syntax, rather than the Common API's callback model.
```

Comment:
Sometimes I write `key:value` because I don't know if this is something that should be translated but I could also just be wrong on that

Reply:
Hmm. I'm not sure. Those aren't actual property names, so I'm inclined to let them be translated.